src/components/main/library/library.js

```diff
@@ -1,7 +1,117 @@
-import React from "react";
+import React, { useState } from "react";
+import { Box, Button, Text, Heading, Container, Flex } from "@chakra-ui/react";
+import {
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  TableCaption,
+  TableContainer,
+} from "@chakra-ui/react";
+import { DeleteIcon } from "@chakra-ui/icons";
+import { v4 as uuidv4 } from "uuid";
 
 const Library = () => {
-  return <div>library</div>;
+  const [books, setBooks] = useState([]);
+
+  const AreBooksEmpty = () => {
+    if (books.length === 0) {
+      return true;
+    }
+    return false;
+  };
```

Comment:
```suggestion
  const AreBooksEmpty = () => books.length === 0;
```
Not required, but you could move this out of the react component itself (can keep it in the same file)
```
const AreBooksEmpty = () => { ... }

const Library = () => { ... }
```